tools/depends/download-files.include

```diff
@@ -49,9 +50,12 @@ ifeq ($(HASH_FOUND),yes)
               echo "download $(ARCHIVE) retry $$tries" ;\
               rm $(TARBALLS_LOCATION)/$(ARCHIVE) ;\
               sleep 3 ;\
-              $(RETRIEVE_TOOL) $(RETRIEVE_TOOL_FLAGS) $(BASE_URL)/$(ARCHIVE) ;\
+              $(RETRIEVE_TOOL) $(RETRIEVE_TOOL_FLAGS) -v $(BASE_URL)/$(ARCHIVE) ;\
               $(HASH_TOOL) $(HASH_TOOL_FLAGS) $(ARCHIVE).$(HASH_TYPE) && exit 0 || tries=$$((tries + 1)) ;\
             done ;\
+            if [ $$tries -gt 3 ] ; then \
```

Comment:
the if doesn't really make sense, if we're here $tries has got to be 4

Reply:
just rm here should be enough